tests/user.test.js

```diff
@@ -0,0 +1,42 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../index';
+import models from '../models';
+
+const { User } = models;
+
+const { should } = chai;
+
+chai.use(chaiHttp);
+
+const googleToken = 'ya29.GlvoBssozS3e4mgyckGahaQKaUqdZ1UUKkfIdRfTc8Eoc9dZM8BX3b2NjPKht2ak8DPdiNBNGJ6tnXEykoqrVdXjYsdD24L8s2xfLfO0nqIs4VTKVSRYhf1viCJb';
+const facebookToken = 'EAAEzaHJ8KTABAO406UiOaJUDIiTEynWbw1slue8tyxTEEqWnaY0U6HJVCKbKtvE5SX0DNFb2ZASUzbJR80wwdbYzeFlD9aKZBlwljykOtwtIepAp1ZBP7uGZByZBRo2XZBSCy20Yafj7AYTuaY3s3L1F3iR43hylZCuMZAZAq0HR4W9FaGWrT3eZCSclwm9uirMUVfqaVCc7jvmtFN0viMk9ZAiDUHSPbSDE82saloGv6c0OgZDZD';
+// testing facebook strategy
```

Comment:
@aaronsekisambu , I see that you are testing both facebook and google strategy but I can see that you mentioned only it is facebook strategy. It would be better to mention it too.

Reply:
Those are your private information. They should not be exposed to the outsiders. 
Access them through the environment variables